hack/wicd-unit.sh

```diff
@@ -18,7 +16,10 @@ if [ -z "$KUBE_SSH_KEY_PATH" ]; then
     return 1
 fi
 
-# Ensure the image used in the release job has ssh installed, as this will be used to remotely run tests against a
-# Windows VM. Running a dummy command to ensure connection to the VM is possible
-ssh -o StrictHostKeyChecking=no -i $KUBE_SSH_KEY_PATH $INSTANCE_USERNAME@$INSTANCE_ADDRESS "dir"
+# Build unit tests for Windows. Specifically targetting packages used by WICD.
+GOOS=windows GOFLAGS=-v go test -c ./pkg/winsvc/... -o wicd_unit.exe
```

Comment:
Consider creating a constant for the binary name `wicd_unit.exe`, so that can be re-used in the script.